mu-find --format=p should output the priority, but it does the path,
while --format=P should output the path, but it does only do P
fix that
Norbert Preining
---
 find/mu-msg-sqlite.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Index: maildir-utils/find/mu-msg-sqlite.c
===================================================================
--- maildir-utils.orig/find/mu-msg-sqlite.c	2009-10-30 03:35:25.000000000 +0900
+++ maildir-utils/find/mu-msg-sqlite.c	2009-10-30 03:35:59.000000000 +0900
@@ -188,7 +188,8 @@
 		mu_msg_sqlite_get_flags (row)); break;
 	case 'd': str = mu_msg_str_date_s(mu_msg_sqlite_get_date(row)); break;
 	case 'z': str = mu_msg_str_size_s(mu_msg_sqlite_get_size(row)); break;
-	case 'p': str = mu_msg_sqlite_get_path(row); break;
+	case 'p': str = mu_msg_str_prio(mu_msg_sqlite_get_priority(row)); break;
+	case 'P': str = mu_msg_sqlite_get_path(row); break;
 	default:
 		buf[0] = fieldchar;
 		return buf;
